server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.ts

```diff
@@ -1003,7 +1003,7 @@ export class AgenticChatController implements ChatHandlers {
 
                 if (toolUse.name === 'fsWrite') {
                     const input = toolUse.input as unknown as FsWriteParams
-                    const document = await this.#triggerContext.getTextDocument(input.path)
+                    const document = await this.#triggerContext.getTextDocumentFromPath(input.path, true, true)
```

Comment:
I think even here we should only load from fs because edits requires fs content, not LSP content